src/app/content/highlights/components/SummaryPopup/HighlightListElement.tsx

```diff
@@ -0,0 +1,128 @@
+import { Highlight, HighlightUpdateColorEnum } from '@openstax/highlighter/dist/api';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import styled, { css } from 'styled-components';
+import { bodyCopyRegularStyle } from '../../../../components/Typography';
+import theme from '../../../../theme';
+import { deleteHighlight, updateHighlight } from '../../actions';
+import { highlightStyles } from '../../constants';
+import { popupBodyPadding } from '../HighlightStyles';
+import ContextMenu from './ContextMenu';
+import HighlightAnnotation from './HighlightAnnotation';
+import HighlightDeleteWrapper from './HighlightDeleteWrapper';
+
+// tslint:disable-next-line:variable-name
+const HighlightOuterWrapper = styled.div`
+  position: relative;
+  overflow: unset;
+
+  :not(:last-child) {
+    border-bottom: solid 0.2rem ${theme.color.neutral.darker};
+  }
+
+  background: ${theme.color.neutral.base};
+`;
+
+// tslint:disable-next-line:variable-name
+const HighlightContent = styled.div`
+  ${bodyCopyRegularStyle}
+  overflow: visible;
+
+  * {
+    overflow: initial;
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+const HighlightContentWrapper = styled.div`
+  padding: 1.2rem ${popupBodyPadding}rem;
+  ${(props: {color: string}) => {
+    const style = highlightStyles.find((search) => search.label === props.color);
+
+    if (!style) {
+      return null;
+    }
+
+    return css`
+      border-left: solid 0.8rem ${style.focused};
+
+      ${HighlightContent} {
+        background-color: ${style.passive};
+      }
+
+      .highlight-note-text {
+        color: ${style.focused};
+      }
+    `;
+  }}
+`;
+
+interface HighlightListElementProps {
+  highlight: Highlight;
+  locationFilterId: string;
+  pageId: string;
+}
+
+// tslint:disable-next-line:variable-name
+const HighlightListElement = ({ highlight, locationFilterId, pageId }: HighlightListElementProps) => {
+  const [isEditing, setIsEditing] = React.useState(false);
+  const [isDeleting, setIsDeleting] = React.useState(false);
+  const dispatch = useDispatch();
+
+  const updateAnnotation = (
+    annotation: string
+  ) => {
+    dispatch(updateHighlight({
+      highlight: {annotation},
+      id: highlight.id,
+    }, {
+      locationFilterId,
+      pageId,
+    }));
+    setIsEditing(false);
+  };
+
+  const updateColor = (color: string) => {
```

Comment:
this color should be `HighlightUpdateColorEnum` or whatever it actually is like `HighlightColorEnum`

Reply:
I've updated ContextMenu and ColorPicker components to use `HighlightColorEnum` so it is more clear conversion